src/app/login/components/Login.tsx

```diff
@@ -142,24 +151,26 @@ export const LoginPage = () => {
                 )}
               />
             </div>
-            <div className="flex items-center justify-between">
-              <Link
-                href="/forgot-password"
-                className="text-sm text-blue-600 hover:underline"
-              >
-                Forgot password?
-              </Link>
-            </div>
+
             <Button
               type="submit"
               className="w-full bg-gray-800"
               data-testId="submitButton"
             >
```

Comment:
can we add button disabled until validation pass ?

Reply:
@Mubashar1009  we have seprate tickets

Reply:
fixed